src/arguments.js

```diff
@@ -137,12 +145,16 @@ const identifyIsFlag = (string, argument) => {
 /**
  * Remove extra enum values (such as 'file', 'path', 'folder', etc.)
  * @param {array} enumArray - array with enum values
- * @param {string} regexp -  regexp for remove
  * @return {array} enumArray - enum without extra values or null if it empty
  */
-const removeExtraEnumValues = (enumArray, regexp) => {
-    const result = enumArray.filter((value) => {
-        return value.toLowerCase().match(regexp) === null;
+const removeExtraEnumValues = (enumArray) => {
+    const extraValues = ['file', 'folder', 'path', 'dir', 'directory', 'files'];
+    const result = enumArray.filter((enumValue) => {
+        let i = 0;
+        extraValues.forEach((extraValue) => {
```

Comment:
I suggest to use `find` instead of `forEach`

Reply:
@svoboda-rabstvo done!